src/Frontend/Page/CreateIdea.hs

```diff
@@ -32,8 +32,8 @@ instance FormPageView PageCreateIdea where
 
     makeForm PageCreateIdea =
         ProtoIdea
-        <$> ("title"         .: DF.text Nothing)
-        <*> ("idea-text"     .: (Markdown <$> DF.text Nothing))
+        <$> ("title"         .: DF.text nil)
```

Comment:
The line is thin and thus I don't feel strongly about it. In a way `nil` is not meant to make you think about which type it is since it is the point of this abstraction. One additional advantage is when a type change `nil` will remain the empty one (lists, maps...).

I revert those.